### DISCUSSION TITLE
Allowing search on exports

### DISCUSSION THREAD

**Discussion by LeahBreedenNyaruka** [nyaruka/rapidpro#4738] [Ideas]
User with large number of messages in 'Sent' folder wants to filter via search query and export results. It is already possible to search messages, at least with text like 'abc' and to match either contact name or body of the message. User needs to be able to: 

1. Write a query like `has "abc" AND urns.tel = "123"` 
2. Export results

**Comment by norkans7** [comment]
Currently we support export message with a label on them.

I think the way to export would be to search the contacts and then label them, and later go to export those messages from the label

Exporting search results is not easy to implement for now
